src/git/from_vcs.py

```diff
@@ -22,8 +22,10 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, runner=run_command):
     version string, meaning we're inside a checked out source tree.
     """
     GITS = ["git"]
+    TAG_PREFIX_REGEX = "*"
     if sys.platform == "win32":
         GITS = ["git.cmd", "git.exe"]
+        TAG_PREFIX_REGEX = "\*" # noqa: W605
```

Comment:
Should just escape the backslash, rather than shield it from QA checks.

```suggestion
        TAG_PREFIX_REGEX = "\\*"
```

Or you could do:

```suggestion
        TAG_PREFIX_REGEX = r"\*"
```

Reply:
I am trying to make sure that linux tests passes. The `flakes` does not like the string `"\*"`. I am trying to make sure that the whole string is passed to command invocation. The raw string is a good idea I didn't think of.